scvi/model/base/_rnamixin.py

```diff
@@ -30,6 +36,95 @@ def _get_transform_batch_gen_kwargs(self, batch):
                 "Transforming batches is not implemented for this model."
             )
 
+    def get_importance_weights(
```

Comment:
why should this be a public method?

the distributions argument is not very clear as well. I'm not sure how anyone would know what to feed here.